### PR TITLE
pcreglow.github.io: journal-week-2

### PR DESCRIPTION
**Pull request by pcreglow**
Created journal-week-2 file

### CODE REVIEW

**Comment by jorgehjr84** [comment]
I really liked your sports analogy. I agree, week 2 beat me up as well, but I think we came out learning a lot. You probably know more than you realize. :+1: for the awesome entry.

File: about/cascading-style-sheets.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
+#(CSS) Cascading Style Sheets
+
+**Display**
+
+```display``` is CSS's most important property for controlling layout. The [```display```](https://developer.mozilla.org/en-US/docs/Web/CSS/display) property specifies the type of rendering box used for an element. ```display``` has an initial value of ```inline``` and applies to all elements. The value ```none``` let's you turn off the display of an element.
```
Comment (al-the-x) [inline about/cascading-style-sheets.md]: Penalty on the play. Excessive use of backticks...

Seriously though, just use _one_ set of backticks when you want code inside of text. Use the triple backticks for fencing off a big block of code.

File: about/cascading-style-sheets.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
+#(CSS) Cascading Style Sheets
+
+**Display**
```
Comment (al-the-x) [inline about/cascading-style-sheets.md]: Maybe use backticks and a heading here? A link to the MDN would be good, too...

> ## [`display`](https://developer.mozilla.org/en-US/docs/Web/CSS/display)

You can extrapolate some of this from the format that I gave you for JavaScript.

File: about/cascading-style-sheets.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+    Syntax examples:
+    
+    display: none;
+    
+    display: inline;
+    display: block;
+    display: contents;
+    display: list-item;
+    display: inline-block;
```
Comment (al-the-x) [inline about/cascading-style-sheets.md]: What about an example of what these do...?

File: about/cascading-style-sheets.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+ 
+*Four* values apply to top, right, bottom and left in that order (clockwise).
+
+    Syntax examples:
+    
+    margin: 20px;
+    margin: 20px 10px;
+    margin: 20px 0 0 20px;
+    margin: auto;
+    margin: 5%;
```
Comment (al-the-x) [inline about/cascading-style-sheets.md]: Great job with these indented blocks for examples... Less copy-pasta from MDN will help you _grok_ the concepts better.

File: journal-week-2.md [hunk 5]
```diff
@@ -0,0 +1,13 @@
...
+
+Those frustrations add a personal element to learning a difficult, challenging subject like JavaScript. I won't deny that it felt like week two at The Iron Yard kicked my ass. Experiencing routine failure has zapped my confidence. On Sunday I had to take a look back and focus on areas where I have made progress instead focusing on negatives. The goal is to stay in the game. Just keep working.
```
Comment (al-the-x) [inline journal-week-2.md]: Keep this attitude. Protect it at all costs. Pepper your learning of new material with review of older material. You'll gain confidence from your mastery, if you remember to celebrate your progress.

**Comment by melrodbos** [comment]
Great final draft Phillip! I think we all feel the sameway about the MVC pattern and how challenging this past week was :thumbsup:

**Comment by Epicurean306** [comment]
Great final draft and loved the sports metaphor: it had me laughing out loud since I think it captures the general feeling of the cohort during and after week two quite well. :+1: 
One thing you may want to do at some point when you get the chance is remove the "rough draft" material or heading (even if you save that content somewhere locally) and do some final revision / polishing into just displaying a clean final draft. :smile:

**Pull request merged**